table/block_based_table_reader.cc

```diff
@@ -696,7 +707,7 @@ SequenceNumber GetGlobalSequenceNumber(const TableProperties& table_properties,
     return kDisableGlobalSequenceNumber;
   }
 
-  SequenceNumber global_seqno = DecodeFixed64(seqno_pos->second.c_str());
+  SequenceNumber global_seqno = largest_seqno;
```

Comment:
Can we still read from `DecodeFixed64(seqno_pos->second.c_str())` and make sure it is either the same as `largest_seqno` or it is not modified?

Reply:
by 'not modified', do you mean verify the checksum?